packages/gatsby-theme/src/pages/blog-page.stories.js

```diff
@@ -0,0 +1,228 @@
+import { useState } from "react"
+import { Box } from "theme-ui"
+import { Section, Image, Text, Link, MobileMenu, Heading } from "ui"
+import ContactBlock from "../blocks/ContactBlock"
+import BlogHeadBlock from "../blocks/BlogHeadBlock"
+import FooterBlock from "../blocks/FooterBlock"
+import HeaderBlock from "../blocks/HeaderBlock"
```

Comment:
We should wait for the #35 PR to be merged before merging this one. Luis added an alias for the blocks.

Reply:
@edvicaty The PR #35  was merged to `master`, so please proceed to update this code.

Reply:
done @georch